README.md

```diff
@@ -17,13 +17,23 @@ Create or modify a configuration file similar to `default.json` in `cm/views`
 ```
 Default Config:
 {
-  // ONLY ONE OF "NODE_IP" or "NODE_DOCKER_HOSTNAME" (offline usage) is required; the former gets precedence
-  "NODE_IP"                 : "0.0.0.0",    // proxy requires ip of a node's rpc server; ie. ipconfig getifaddr en0
-  "NODE_DOCKER_HOSTNAME"    : "dappley"     // hostname of the node's docker container running on the default network bridge
-  "NODE_RPC_PORT"           : 50050,        // port # of a node's rpc server
-  "WEB_SERVER_PORT"         : 8080,         // webserver runs at http://0.0.0.0:8080
-  "POLLING_INTERVAL"        : 5000,         // interval in milliseconds at which to poll a node's rpc server
-  "GRPC_PROXY_PORT"         : 8081          // port # of grpc-proxy for webserver to communicate with node
+  // ONLY ONE OF "NODE.IP" or "NODE.DOCKER_HOSTNAME / NODE.DOCKER_NETWORK" (offline usage) is required; the former gets precedence
+  "NODE": {
+    "DOCKER_HOSTNAME"  : "dappley",            // hostname of the node's docker container
+    "DOCKER_NETWORK"   : "go-dappley_default"  // the network the docker container is connected to
+    "IP"               : "0.0.0.0",            // proxy requires ip of a node's rpc server; ie. ipconfig getifaddr en0
+    "RPC_PORT"         : 50050,                // port # of a node's rpc server
+  },
+  "AUTH_SERVER": {
```

Comment:
`"DOCKER_HOSTNAME": "auth-server",` should be in `AUTH_SERVER` as default config? as I see it is included in `default.json` in `cm`. Also, the instruction after this also asked the user to run `auth-server` in docker.

Also wanna know why does `DOCKER_NETWORK` not exist in `AUTH_SERVER`? I thought they have to be run in the same network in docker?

Reply:
Thanks for catching that, I don't actually use `AUTH_SERVER.DOCKER_HOSTNAME` and I don't think the host name needs to be configurable so I'll just remove this from `default.json`.

`DOCKER_NETWORK` is used by `dappley-stats` to join the same local docker network that `go-dappley` is running on, therefore it's really just tied to the docker build hence, it's only used [here](https://github.com/dappley/dappley-stats/blob/2679aded39e062b6ab8e297bbc0285e7f8a1ee9b/cm/templates/docker-compose.yml.mustache#L43-L48).

Reply:
0e2bc68